homeassistant/components/nest/config_flow.py

```diff
@@ -235,7 +236,7 @@ async def async_step_import(self, info):
         config_path = info["nest_conf_path"]
 
         if not await self.hass.async_add_executor_job(os.path.isfile, config_path):
-            self.flow_impl = DOMAIN
+            self.flow_impl = DOMAIN  # type: ignore
```

Comment:
Why is this ignored?

Reply:
I ignored this following the `#type: ignore` in the parent.  https://github.com/home-assistant/core/blob/391b2f8ccd67d3e99ac24bfa0af36de2d0532540/homeassistant/helpers/config_entry_oauth2_flow.py#L224 -- the type checker seemed to want to treat this as the abstract oauth class, but this code is assigning a string, which doesn't match. Since its for the legacy nest integration, I punted.

I'm happy to improve the typing here, though I need some guidance.